pkg/ignition/ignition.go

```diff
@@ -12,25 +12,21 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Added to manage ignition versions in one file
-var EmptyIgnitionConfig = types.IgnitionConfig{}
-var EmptyIgnition = types.Config{}
-
-//go:generate mockgen -source=ignition.go -package=ignition -destination=mock_ignition.go
-type Ignition interface {
+//go:generate mockgen -source=Ignition.go -package=Ignition -destination=mock_ignition.go
+type IgnitionInterface interface {
```

Comment:
the naming is a bit weird:/ do we have to export the struct? we don't export in coreos.go for example, or in [release.go](https://github.com/openshift/assisted-service/blob/c4a6395c0059890d083379e0bc28f34e072f669d/internal/oc/release.go#L58-L59)